packages/component-library/styles/responsive.scss

```diff
@@ -1,6 +1,13 @@
+$ca-breakpoint-mobile-sm: 481px;
 $ca-breakpoint-mobile: 768px;
 $ca-breakpoint-tablet: 1024px;
 
+@mixin ca-media-mobile-sm {
```

Comment:
FYI all of these `component-library/styles/*` files are deprecated, we're wanting to move away from them.

The current recommended way to do media queries is to use the `$layout-breakpoints-` design tokens (we don't offer mixins anymore).

@lloydstubber do we want to add a `small` breakpoint to our design tokens instead of doing this?

Reply:
Spoke to @katsambarcus and 481px isn't our small breakpoint for mobiles (it's 375px) so I think we'll just need to keep this in here @hqta @dougmacknz